permissions/plugin-anything-goes-formatter.yml

```diff
@@ -6,6 +6,5 @@ issues:
 paths:
   - "org/jenkins-ci/plugins/anything-goes-formatter"
 developers: []
-#CD blocked for lack of maintainers
-#cd:
-#  enabled: true
+cd:
+  enabled: true
```

Comment:
this doesn't really seem valid to me, isn't it clearer if we error if there's no maintainers?